packages/gestalt/src/TableSortableHeaderCell.js

```diff
@@ -38,33 +38,36 @@ export default function TableSortableHeaderCell(props: Props) {
 
   return (
     <TableHeaderCell colSpan={colSpan} rowSpan={rowSpan} scope={scope}>
-      <Touchable
-        onTouch={onSortChange}
-        onMouseEnter={() => setHovered(true)}
-        onMouseLeave={() => setHovered(false)}
-        onFocus={() => setFocused(true)}
-        onBlur={() => setFocused(false)}
-      >
-        <Box display="flex">
-          {children}
-          <Box
-            marginLeft={2}
-            dangerouslySetInlineStyle={{
-              __style: { visibility },
-            }}
-          >
-            <Icon
-              accessibilityLabel=""
-              icon={
-                status === 'active' && sortOrder === 'asc'
-                  ? 'sort-ascending'
-                  : 'sort-descending'
-              }
-              color={status === 'active' ? 'darkGray' : 'gray'}
-            />
+      <Box width="fit-content">
+        <Touchable
+          fullWidth={false}
+          onTouch={onSortChange}
+          onMouseEnter={() => setHovered(true)}
+          onMouseLeave={() => setHovered(false)}
+          onFocus={() => setFocused(true)}
+          onBlur={() => setFocused(false)}
+        >
+          <Box display="flex">
```

Comment:
@PayalModi can we center align the items? Looks like design also wants this:

Figma:
![image](https://user-images.githubusercontent.com/127199/83815633-799df180-a675-11ea-9660-fbc0ba54544c.png)

Before:
![image](https://user-images.githubusercontent.com/127199/83815657-89b5d100-a675-11ea-9ef3-8ff4c5002cef.png)

After:
![image](https://user-images.githubusercontent.com/127199/83815731-bc5fc980-a675-11ea-9e89-f33a591ce8b7.png)